.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: cpp
+
+sudo: required
+
+dist: focal
+
+matrix:
+  include:
+    - os: linux
+      dist: bionic
+      compiler: clang
```

Comment:
Why choose clang compiler? As I can see, we usually use gcc.
Maybe we can specify a "compiler" on one level with "language"?

Reply:
> Why choose clang compiler?

Sometimes compilation fails with gcc, for example [here](https://travis-ci.org/github/tarantool/mysql-tarantool-replication/jobs/735535579):
```
/home/travis/build/tarantool/mysql-tarantool-replication/lib/libslave/field.h:176:8: error: ‘vector’ in namespace ‘std’ does not name a template type
  176 |   std::vector<std::string> str_values;
      |        ^~~~~~
/home/travis/build/tarantool/mysql-tarantool-replication/lib/libslave/field.h:25:1: note: ‘std::vector’ is defined in header ‘<vector>’; did you forget to ‘#include <vector>’?
   24 | #include <mysql/binary_log_funcs.h>
  +++ |+#include <vector>
   25 | /*
```
It is a known problem and described in tutorial included to Tarantool documentation - https://www.tarantool.io/en/doc/latest/tutorials/libslave/

This time I want just to enable CI in a project so I left clang only.

> As I can see, we usually use gcc.

It's better to use both if there are no serious reasons to do not so.
 
> Maybe we can specify a "compiler" on one level with "language"?

No sense to do it as we specify compiler in a matrix.